packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -2,9 +2,6 @@ import { autorun } from 'mobx'
 import { addDisposer, getRoot, isValidReference, types } from 'mobx-state-tree'
 
 import Step from './Step'
-import taskRegistry, { taskModels } from '@plugins/tasks'
-
-const taskTypes = types.union(...taskModels)
```

Comment:
Is this simply cruft that was left in this component?

Reply:
I think so.